PendingReleaseNotes

```diff
@@ -18,3 +18,6 @@
 * The JSON output of the ``osd find`` command has replaced the ``ip``
   field with an ``addrs`` section to reflect that OSDs may bind to
   multiple addresses.
+
+* CephFS clients without the 's' flag in their authentication capability
+  string will no longer be able to create/delete snapshots.
```

Comment:
Let's add to this note what command needs to be run to enable snapshots for an existing credential.